defold-spine/src/script_spine.cpp

```diff
@@ -679,6 +679,47 @@ namespace dmSpine
         return 0;
     }
 
+    /*# Set world space bone position
+     * Note that bones positions will be overwritten by active animations and constraints(IK, Path, Transform...), only change positions of bones that is not affected by those.
+     *
+     * @name spine.get_go
+     * @param url [type:string|hash|url] the spine model to query
+     * @param bone_id [type:string|hash] id of the corresponding bone
+     * @param position [type:vector3] position in world space
+     * @examples
+     *
+     * The following examples assumes that the spine model has id "spinemodel" and it has bone named "crosshair"
+     *
+     * How to set bone position to mouse position("crosshair" could be IK target for example):
+     *
+     * ```lua
+     * function on_input(self)
+     *   if action_id == nil then
+     *     spine.set_bone_position("#spinemodel", "crosshair", vmath.vector3(action.x, action.y, 0))
+     *   end
+     * end
+     * ```
+     */
+    static int SpineComp_SetBonePosition(lua_State* L)
+    {
+        DM_LUA_STACK_CHECK(L, 0);
+
+        SpineModelComponent* component = 0;
+        dmMessage::URL receiver; // needed for error output
+        dmGameObject::GetComponentFromLua(L, 1, SPINE_MODEL_EXT, 0, (void**)&component, &receiver);
+
+        dmhash_t bone_id = dmScript::CheckHashOrString(L, 2);
+        Vectormath::Aos::Vector3* position = dmScript::CheckVector3(L, 3);
```

Comment:
We only do casts if we know it's the correct type.
Here, they're not the same: `Vector3 != Point3`

You can create a Point3 like so: `Point3(v3)`